charts/lakefs/values.yaml

```diff
@@ -92,3 +92,7 @@ extraEnvVars:
 #   successThreshold: 4
 #   timeoutSeconds: 1
 #   initialDelaySeconds: 5
+
+# Everything under fluffy is lakeFS enterprise only.
+# fluffy:
+#   enabled: false
```

Comment:
Think we will require documenting the enterprise feature in the docs site - on how to enable it in the charts.